src/static/store.json

```diff
@@ -6,6 +6,9 @@
       },
       {
         "barcode": "STO-003D"
+      },
+      {
+        "barcode": "STO-12DC"
```

Comment:
Is it STO-12CD instead of STO-12DC?
In sql , it is as follows
"_INSERT INTO location
  (barcode, name)
VALUES
  (**'STO-12CD', 'Dessicator'**_"  And 'Rack 1' is STO-12DC

Reply:
Well spotted! Thanks